app/i18n/en/feedback.php

```diff
@@ -101,6 +101,10 @@
 			'_' => 'User %s has been deleted',
 			'error' => 'User %s cannot be deleted',
 		),
+		'updated' => array(
+			'_' => 'The user %s has been updated',
```

Comment:
For consistency with the other strings (see above) that should be "User %s…"

Reply:
Done